sorc/chgres_cube.fd/CMakeLists.txt

```diff
@@ -24,6 +24,11 @@ elseif(CMAKE_Fortran_COMPILER_ID MATCHES "^(GNU)$")
   set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -ffree-line-length-0 -fdefault-real-8 -fconvert=big-endian")
 endif()
 
+# Turn on this argument mismatch flag for gfortran10.
```

Comment:
This needs to go in side the check for the GNU compiler. You don't want to add those options for any compiler.

Reply:
Duh. I have fixed.